### ISSUE TITLE
invalid image 

### ISSUE THREAD

**Issue by castelblends** [marcofugaro/three-projected-material#9]
Hi, I have just found your library and I'm testing it. 

I'm getting a "not valid image texture" message in my browser's console and I wonder if the only way to load textures is by using the asset manager under ./lib. ?

I'm creating a texture unsing 

const texture = new THREE.TextureLoader().load( 'textures/land_ocean_ice_cloud_2048.jpg' );

Just like the documentation example from threejs.

Any light over this soul would be appreciated.

Thanks.

**Comment by marcofugaro** [comment]
Yeah it should work fine with normal textures, the `AssetManager` is just an util I'm using in the examples.

Does the texture work if you pass it as a `map` to a normal material first? Maybe the texture can't be found in your project.

**Comment by castelblends** [comment]
Yes, the texture works fine  as you can see here...

![Screenshot from 2020-11-11 12-17-04](https://user-images.githubusercontent.com/61828683/98836325-efbe3580-2417-11eb-93f1-605ae4ddaf38.png)

first....


_var texture2= new THREE.TextureLoader().load('./assets/forest.jpg');
const geometry = new THREE.BoxGeometry(1, 1, 1);
const mta= new THREE.MeshBasicMaterial({map:texture2});
var cubo = new THREE.Mesh(geometry,mta);_


then....

_camera2 = new THREE.PerspectiveCamera(45, 1, 0.01, 3)
camera2.position.set(-1, 1.2, 1.5)
camera2.lookAt(0, 0, 0)_	



and then...


const material = new ProjectedMaterial({
		  camera2, // the camera that acts as a projector
		  texture2, // the texture being projected
		  color: '#cccccc', // the color of the object if it's not projected on
		  textureScale: 0.8, // scale down the texture a bit
		  cover: true, // enable background-size: cover behaviour, by default it's like background-size: contain
		})_



and while loading texture2 is where I get this....

![Screenshot from 2020-11-11 12-20-19](https://user-images.githubusercontent.com/61828683/98836781-7b37c680-2418-11eb-824a-d941df5cc7b0.png)

**Comment by marcofugaro** [comment]
@castelblends I see your problem.

while passing `camera2` in the object, it gets translated as `camera2: camera2`. This is [ES6 Object property shorthand](https://alligator.io/js/object-property-shorthand-es6/).

This is the correct way of doing your example:

```js
const material = new ProjectedMaterial({
  camera: camera2, // the camera that acts as a projector
  texture: texture2, // the texture being projected
  color: '#cccccc', // the color of the object if it's not projected on
  textureScale: 0.8, // scale down the texture a bit
  cover: true, // enable background-size: cover behaviour, by default it's like background-size: contain
})
```

**Comment by castelblends** [comment]
ok, applied those changes, then I got "texture.image is undefined"

so I loaded and assigned an image to the texture like this:

	const ima= new THREE.ImageLoader().load('./assets/forest.jpg');
	var texture2= new THREE.TextureLoader().load('./assets/forest.jpg');
	texture2.image=ima;

and got kicked back with this:

![Screenshot from 2020-11-11 18-55-08](https://user-images.githubusercontent.com/61828683/98873869-835f2880-244f-11eb-9344-f70dae00b204.png)

**Comment by marcofugaro** [comment]
@castelblends sure! I'll do an example without AssetManager soon, when I have time.

You don't need to use the ImageLoader, you just do:

```js
var texture2 = new THREE.TextureLoader().load('./assets/forest.jpg');
```

**Comment by marcofugaro** [comment]
Fixed in `v2.0.0`. The examples have been refactored and `AssetManager` has been removed.

Check them out: https://marcofugaro.github.io/three-projected-material/
